daml-lf/archive/da/daml_lf_1.proto

```diff
@@ -748,6 +762,17 @@ message CaseAlt {
     string binder = 3;
   }
 
+  // Enum pattern
+  message Enum {
+
+    // name of the type constructor
+    TypeConName con = 1;
+
+    // name of the variant constructor
+    // *Must be a valid identifier*
+    string variant = 2;
```

Comment:
i would not call this "variant", to avoid confusion with... variants.

* Java calls them "constants": https://docs.oracle.com/javase/specs/jls/se12/html/jls-8.html#jls-EnumBody
* C# calls them "members": https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/enums#enum-members
* C++ calls them "definitions": https://en.cppreference.com/w/cpp/language/enum

None of them are that great... I'd personally be fine with "name" or "tag" or something of the sorts

@hurryabit ideas?

Reply:
use "constructor" everywhere